src/ansys/units/quantity.py

```diff
@@ -94,7 +55,15 @@ def __init__(
                 units = copy_from.units
                 value = copy_from.value
 
-        self._value = float(value)
+        if isinstance(value, list):
+            try:
+                import numpy as np
+
+                self._value = np.array(value)
+            except:
+                raise QuantityError.REQUIRES_NUMPY()
+        else:
+            self._value = value
```

Comment:
Just something to be aware of is that we are referencing the external `array` but copying the external `list`. So slightly different behaviours in each case (if anything changes).

Reply:
...to see how that compares with pint, you could instantiate a pint `Quantity` from an `array`, change the source data and see how the `Quantity` data is affected.

Reply:
This is pint: 
```
from pint import UnitRegistry
import numpy as np

ureg = UnitRegistry()
q = ureg.Quantity

arr = np.array([1,2,7])
q1 = q(arr, ureg.meter)

arr[0] = 10
print(q1) # [10  2  7] meter
```
Should we operate the same?

Reply:
I would say that looks pretty reasonable, so yes.